### PR TITLE
Include e2e build tag into linter.

### PR DESCRIPTION
**Pull request by StefanHauth**
# Description

Build tag e2e is missing for linter run in pre-commit hook, which leads to problems when committing. Added it to pre-commit hook.

## How can this be tested?
Commit changes to a branch.

## Checklist
- [n/a] Unit tests have been updated/added
- [x] PR is labeled accordingly
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: .github/pre-commit [hunk 1]
```diff
@@ -38,7 +38,7 @@ else
     done
 fi
 
-golangci-lint run --build-tags integration,containers_image_storage_stub
+golangci-lint run --build-tags integration,containers_image_storage_stub,e2e
```
Comment (chrismuellner) [inline .github/pre-commit:41]: instead of duplicating the commands, we should use `make go/lint` and `make go/test` respectively

Comment (chrismuellner) [reply .github/pre-commit:41]: this file is pretty out of date 🙂

**Review by chrismuellner — DISMISSED** [review]
makefile commands

**Review by luhi-DT — APPROVED** [review]

File: .github/pre-commit (hunk 1 above)
Comment (StefanHauth) [reply .github/pre-commit:41]: we will do the rework in a separate PR but get this out of the door now as it causes some considerable friction in day-to-day dev work

**Pull request merged**